pkg/api/secretbootstrap/secretboostrap.go

```diff
@@ -17,10 +18,27 @@ const (
 )
 
 type BitWardenContext struct {
-	BWItem     string        `json:"bw_item"`
-	Field      string        `json:"field,omitempty"`
-	Attachment string        `json:"attachment,omitempty"`
-	Attribute  AttributeType `json:"attribute,omitempty"`
+	BWItem               string                 `json:"bw_item"`
+	Field                string                 `json:"field,omitempty"`
+	Attachment           string                 `json:"attachment,omitempty"`
+	Attribute            AttributeType          `json:"attribute,omitempty"`
+	DockerConfigJSONData []DockerConfigJSONData `json:"dockerconfigJSON,omitempty"`
+}
+
+type DockerConfigJSONData struct {
+	BWItem                    string `json:"bw_item"`
+	RegistryURLBitwardenField string `json:"registry_url"`
```

Comment:
Please also update the json tags

Reply:
Shall we change them to `registry_url_bw_field` `auth_bw_field` and `email_bw_field` ?